### PR TITLE
tox.ini: use testpaths, cleanup other pytest options

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -201,13 +201,13 @@ commands = python scripts/release.py {posargs}
 
 [pytest]
 minversion = 2.0
-plugins = pytester
```
Comment (blueyed) [inline tox.ini]: This was added in b6ec5a57, but appears to never has been supported?!

Comment (nicoddemus) [reply tox.ini]: Hmm yeah, I can't find any code that supports this

**Comment by nicoddemus** [comment]
Thanks!

File: tox.ini [hunk 2]
```diff
@@ -201,13 +201,13 @@ commands = python scripts/release.py {posargs}
...
-addopts = -ra -p pytester --ignore=testing/cx_freeze
```
Comment (RonnyPfannschmidt) [inline tox.ini]: no cx_freeze anymore?

Comment (nicoddemus) [reply tox.ini]: Oh it is OK, we have an `freeze` testenv dedicated to it. 👍

Comment (RonnyPfannschmidt) [reply tox.ini]: @nicoddemus but we no longer ignore it now

Comment (nicoddemus) [reply tox.ini]: That's OK, it contains only two dummy tests: https://github.com/pytest-dev/pytest/tree/master/testing/freeze/tests

We could perhaps rename them to something else that's not picked up by pytest by default, but not sure it is worth the trouble. 😁

Comment (blueyed) [reply tox.ini]: AFAIK the `cx_freeze` gets generated now.
`cx_freeze` would not have matches `freeze` currently anyway.

**Pull request merged**